Components/sideBar.css

```diff
@@ -0,0 +1,26 @@
+.side-container{
+    padding: 1rem;
+    width: 20%;
+    position: sticky;
+    top: 0rem;
```

Comment:
instead of writing 0rem, you can write only 0 here.

Reply:
Thanks, done